datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -236,6 +236,12 @@ def _get_metric_limit(self):
         try:
             limit = int(raw_limit)
         except (ValueError, TypeError):
+            self.warning(
```

Comment:
```suggestion
            self.log.warning(
```

`self.warning` will make the message appear in DataDog UI. Is that intended?

Reply:
Yes, I think it's sensible to have this error displayed in the host logs on the app, or customers probably won't see it.